src/meta/buildTags.tsx

```diff
@@ -658,6 +658,12 @@ const buildTags = (config: BuildTagsParams) => {
     });
   }
 
+  if (config.additionalLinkTags?.length) {
+    config.additionalLinkTags.forEach((tag, index) => {
+      tagsToRender.push(<link key={`link${index}`} {...tag} />);
```

Comment:
I am not sure this will work like this 🤔 

Can you test the following. On the index page at 4 addtionalLinkTags. Then on the override page can you add one more tag? You should see 5 tags in the head but I suspect that they will be only 4.

Reply:
@garmeeh 
Hey, thanks for the review!
I added a test case for the override.

It has 2 apple-touch-icons (76x76 and 120x120) on the index page as default SEO, and I added an another apple-touch-icon (180x180) on the override page.  Then I added an additional test to make sure there are three apple-touch-icon head on override page.

Is this the right test case that you want me to cover it?